src/rpc/account.ts

```diff
@@ -200,20 +199,24 @@ export class AccountRpc {
             tx.setFee(fee);
         }
         if (tx.fee() == null) {
-            tx.setFee(getMinimumFee(tx));
+            const minFee = await this.rpc.chain.getMinTransactionFee(tx.type());
```

Comment:
The minimum fee could be different according to the block number. Please pass the block number.